recipes/pymedphys/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pymedphys" %}
+{% set version = "0.1.9" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d7890a1e26d912ee9c219ef17205f8b6a84d5ae403c7367e5f59fa708699a939" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - pandas
+    - matplotlib
+    - numba
+    - attrs
+    - psutil
+    - pymssql
+    - keyring
+    - shapely
+    - pydicom
+
+test:
+  imports:
+    - pymedphys
+    - pymedphys.level1
+    - pymedphys.level2
+
+about:
+  home: The package home page
```

Comment:
Can you add the home page in here?

Reply:
Certainly, just the GitHub repo address?